build.cake

```diff
@@ -37,6 +40,20 @@ var srcProjectNames = new[]
     "MongoDB.Driver.GridFS"
 };
 
+public static bool IsSuiteRunnable(string framework)
```

Comment:
OCSP was only supported starting in .NET Core 3.0 on Linux, so we don't run the tests if we're testing OCSP on a lower version on Linux